frontend/src/lib/rbac-util.ts

```diff
@@ -105,17 +105,27 @@ export async function checkAdminAccess() {
 }
 
 type Verb = 'get' | 'patch' | 'create' | 'delete' | 'update'
-
-export function rbacResource(verb: Verb, resource: IResource, namespace?: string, name?: string) {
+type SubResource = 'join'
```

Comment:
we will only need join?

Reply:
Currently this is the only subresource I know of; and that we currently will use I guess.  Are there more?

Reply:
no, just feel this code is a little bit confusing, but I'm good with that 👍